pandas/tests/frame/methods/test_convert_dtypes.py

```diff
@@ -26,3 +26,8 @@ def test_convert_dtypes(self, convert_integer, expected):
             }
         )
         tm.assert_frame_equal(result, expected)
+
+    def test_convert_empty(self):
+        # Empty DataFrame can pass convert_dtypes, see GH#40393
+        df = pd.DataFrame().convert_dtypes()
+        assert df.empty
```

Comment:
I think it'd be better to use tm.assert_frame_equal, e.g. testing dtypes are the same.

Reply:
Done, yes I noticed I noticed the pandas testing asserts seemed to be used exclusively. I wasn't quite sure which way to use them but hopefully this change looks okay.